guides/deployment/heroku.md

```diff
@@ -144,6 +144,7 @@ Then, we'll add the production database configuration to `config/prod.exs`:
 ```elixir
 # Configure your database
 config :hello, Hello.Repo,
+  adapter: Ecto.Adapters.Postgres,
```

Comment:
I don't think the adapter is required in the config going forward, it should be specified in the `MyApp.Repo` module instead.

https://github.com/phoenixframework/phoenix/blob/473a3cb40aae1ba36c70dfc4841ace2d3aab7471/installer/templates/phx_ecto/repo.ex#L4